recipes/pytorch-3dunet/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.7.0" %}
+
+package:
+  name: pytorch-3dunet
+  version: {{ version }}
+
+source:
+  url: https://github.com/wolny/pytorch-3dunet/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 20bff5c53404d6b34b02a6a051cad80d93bc57646a76117e917951d34d2780dd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - predict3dunet = pytorch3dunet.predict:main
+    - train3dunet = pytorch3dunet.train:main
+
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - tensorboard
+    - tqdm
+    - h5py
+    - scipy
+    - scikit-image
+    - pyyaml
+  run_constrained:
+    - pytorch >=1.7.0
+
+test:
```

Comment:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

Reply:
Previously, I've had `pytest tests` execution in here, but removed it due to build failing on win 64 (see my comment below).

Reply:
Please add the pip check, we don't need pytest but `pip check` is essential for Python packages.